CMakeLists.txt

```diff
@@ -1,22 +1,22 @@
-# Set the minimum cmake version requires by this project.
-cmake_minimum_required(VERSION VERSION 3.18 FATAL_ERROR)
+# Minimum CMake version 3.18 for CUDA --std=c++17 
+cmake_minimum_required(VERSION 3.18...3.25 FATAL_ERROR)
 
-# Name the project and set languages
-project(example CUDA CXX)
-
-# Option to enable/disable building the visualisation
-option(VISUALISATION "Enable visualisation support" OFF)
```

Comment:
Is there a reason you've removed this?

Thought we agreed yesterday that it makes sense to allow users to change the default value here (but not in examples)? Likewise, this should allow users to specify vis before flamegpu2 is fetched, avoiding double config (if using CMake GUI).

Reply:
I'd just copy pasted in from an example and didn't re-add it apparently, pushed with it reintroduced and manually tested that defaulting it to ON works.